### PR TITLE
1 hw RESTapi

### PR DESCRIPTION
**Pull request by SlikeX**

### CODE REVIEW

**Comment by devbarden** [comment]
Add .gitignore file because node modules is in your repository. Thats bad. Because its very heavyweight. Please read more about it

**Comment by devbarden** [comment]
I can't review it yet because as you can see there is 1138 files changes in PR. And that is why because you didn't add .gitignore file with node_modules

**Comment by devbarden** [comment]
So please ping me when you fix it

File: routes/accounts.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+
+
+const accounts = [
```
Comment (devbarden) [inline routes/accounts.js:5]: The Data should store is json file

File: routes/categories.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  'sport',
+  'entertament'
+]
+
+router.get('/',(request,response) => {
+  response.send(categories)
+});
+
+router.get('/:id',(request,response) => {
+  console.log(categories.length == 0)
```
Comment (devbarden) [inline routes/categories.js]: please remove console.log. Its good to use in development but not in production

**Review by devbarden — COMMENTED** [review]
Not bad but please keep your code in one style. As I can see sometimes you add `;` in the end of the line and sometimes no. Also sometimes you add some extra empty line. Its minor issue but will be better if you will keep coding in one style

**Comment by SlikeX** [comment]
@devbarden Could I keep accounts in JS file, coz in the next HW we're gonna transfer it to DB and it won't be any data in this file?

**Comment by devbarden** [comment]
> @devbarden Could I keep accounts in JS file, coz in the next HW we're gonna transfer it to DB and it won't be any data in this file?

sure, no problem

**Comment by devbarden** [comment]
if you need you can merge it

**Pull request merged**